watertap/costing/unit_models/surrogate_crystallizer.py

```diff
@@ -0,0 +1,189 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2024, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pyomo.environ as pyo
+from watertap.costing.util import (
+    register_costing_parameter_block,
+    make_capital_cost_var,
+)
+
+
+def build_surrogate_crystallizer_cost_param_block(blk):
+
+    blk.steam_pressure = pyo.Var(
+        initialize=3,
+        units=pyo.units.bar,
+        doc="Steam pressure (gauge) for crystallizer heating: 3 bar default based on Dutta example",
+    )
+
+    blk.efficiency_pump = pyo.Var(
+        initialize=0.7,
+        units=pyo.units.dimensionless,
+        doc="Crystallizer pump efficiency - assumed",
+    )
+
+    blk.pump_head_height = pyo.Var(
+        initialize=1,
+        units=pyo.units.m,
+        doc="Crystallizer pump head height -  assumed, unvalidated",
+    )
```

Comment:
Just a note for everyone's awareness: The crystallizer cost for steam (including steam properties) and the pump is being removed in PR #1482 . These costs are now broken down into the steam heater (which uses the water vapor property package) and the pump. As a result, steam and pump flows are calculated at the flowsheet level, with their respective costs corresponding to the unit models.

Reply:
Good to know--will leave here in this PR and let your PR take care of this.